uclalib_vmware_snapshot.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+cd /etc/ansible || exit 1
+
+limit="$GROUP"
+if [[ "$GROUP" == "SPECIFY" ]]; then
+  if [[ -z $HOSTS ]]; then
+    echo "Error in build parameters: HOSTS must be specified"
+    exit 1
+  fi
+  limit="$HOSTS"
+elif [[ -n $HOSTS ]]; then
+  echo "Error in build paramerters: HOSTS must be blank"
+  echo "or GROUP must be SPECIFY"
+  exit 1
+fi
+
+ansible-playbook \
+  plays/uclalib_vmware_snapshot.yml \
+  -l "$limit" \
+  -e remove_snapshot="$REMOVE_SNAPS" \
+  -e days_to_keep_snapshots="${AGE:-7}" \
```

Comment:
Curious, what does the -7 represent?

Reply:
${a:-string} is a sh expansion that means "default value". if "a" is set and non-empty, then it expands to the value of a, otherwise it expands to "string" in this example.

In this case, the default value is "7" which matches the value in the playbook, so it's redundant, but removing it from the playbook would cause the playbook to fail if omitted, and other "add only if present" processing is IMHO unnecessary in this case.

```bash
bash-3.2$ echo ${a:-string}
string
bash-3.2$ a=apples
bash-3.2$ echo ${a:-string}
apples
bash-3.2$ a=""
bash-3.2$ echo ${a:-string}
string
bash-3.2$
```

Reply:
Ah! that's handy. I learned something new :) Thank you for the explanation

Reply:
Or, in short, at least a week old by default :D